package.json

```diff
@@ -110,6 +110,7 @@
     "prettier": "1.18.2",
     "puppeteer": "1.17.0",
     "rcfile": "1.0.3",
+    "react-testing-library": "7.0.1",
```

Comment:
FYI: I moved the rtl dep to the root package. Only a couple of packages (e.g. app-shell, jest preset) define it explicitly as a peer dependency.